src/main/webapp/dev/js/common/instructorEnroll.js

```diff
@@ -76,53 +76,50 @@ function toggleChevronImage(panelCollapse, toggleChevron) {
  * Expands panel, showing the spreadsheet interface.
  */
 function expandStudentsPanel(panelCollapse) {
-    $(panelCollapse[0]).collapse('show');
-    $(panelCollapse[0]).addClass('checked');
+    panelCollapse.collapse('show');
 }
 
 /**
  * Collapses panel, hiding the spreadsheet interface.
  */
 function collapseStudentsPanel(panelCollapse) {
-    $(panelCollapse[0]).collapse('hide');
-    $(panelCollapse[0]).removeClass('checked');
+    panelCollapse.collapse('hide');
 }
 
 /**
  * Expands/Collapses the panel depending on the current state of the panel.
  */
 function toggleStudentsPanel($panelHeading, panelCollapse, displayIcon, toggleChevron) {
+    let isPanelExpanded = false;
     displayIcon.html('');
-    if ($(panelCollapse[0]).hasClass('checked')) { // panel is shown
+    if (panelCollapse.hasClass('in')) { // panel is shown
         collapseStudentsPanel(panelCollapse);
     } else {
         expandStudentsPanel(panelCollapse);
+        isPanelExpanded = true;
     }
-    toggleChevronImage(panelCollapse, toggleChevron);
+    toggleChevronImage(isPanelExpanded, toggleChevron);
+    return isPanelExpanded;
 }
 
 /**
  * Displays a message informing the user that there are no existing students in the course.
  */
-function displayNoExistingStudents(displayIcon) {
-    const statusMsg = `
-        <strong style="margin-left: 1em; margin-right: 1em;">
-            [ No existing students in course. ]
-        </strong>`;
-    displayIcon.html(statusMsg);
+function displayNoExistingStudents(displayStatus) {
+    displayStatus.html('<div class="ajax-error-message">'
+            + '<strong>[ No existing students in course. ]</strong>'
+            + '</div>');
 }
 
 /**
  * Displays a message informing the user that an error surfaced during the AJAX request.
  * @param displayIcon
  */
-function displayErrorExecutingAjax(displayIcon) {
-    const warningSign = '<span class="glyphicon glyphicon-warning-sign"></span>';
-    const errorMsg = `
-        <strong style="margin-left: 1em; margin-right: 1em;">
-            [ Failed to load. Click here to retry. ]
-        </strong>`;
-    displayIcon.html(warningSign + errorMsg);
+function displayErrorExecutingAjax(displayStatus) {
+    displayStatus.html('<div class="ajax-error-message">'
+            + '<span class="glyphicon glyphicon-warning-sign"></span>'
+            + ' <strong>[ Failed to load. Click here to retry. ]</strong>'
+            + '</div>');
```

Comment:
Mixing HTML inside Javascript is also not a good practice. You should add the
```
<div class="ajax-error-message">
<span class="glyphicon glyphicon-warning-sign"></span>
</div>
```
to the JSP and then modify the text content and classes accordingly. This also applies to other display status functions.